buildkite/bazelci.py

```diff
@@ -41,13 +41,45 @@
 # Initialize the random number generator.
 random.seed()
 
-CLOUD_PROJECT = (
-    "bazel-public"
-    if os.environ.get("BUILDKITE_ORGANIZATION_SLUG") == "bazel-trusted"
-    else "bazel-untrusted"
-)
-
-DOWNSTREAM_PROJECTS = {
+BUILDKITE_ORG = os.environ["BUILDKITE_ORGANIZATION_SLUG"]
+THIS_IS_PRODUCTION = BUILDKITE_ORG == "bazel-untrusted"
+THIS_IS_TESTING = BUILDKITE_ORG == "bazel-testing"
+THIS_IS_TRUSTED = BUILDKITE_ORG == "bazel-trusted"
+THIS_IS_SPARTA = True
+
+CLOUD_PROJECT = "bazel-public" if THIS_IS_TRUSTED else "bazel-untrusted"
```

Comment:
No dedicated GCP project for testing?

Reply:
No, I didn't want to create yet another project, acquire quota, ...

But I setup the service account buildkite-testing@ so that it doesn't have access to anything except the dedicated test resources (like GCS buckets). :)